src/main/java/io/antmedia/enterprise/streamapp/StreamApplication.java

```diff
@@ -31,9 +35,13 @@
 	
 	@Override
 	public boolean appStart(IScope app) {
-		WebSocketLocalHandler.app = this;
 		if(io.antmedia.rest.BroadcastRestService.isEnterprise()) {
-			appAdaptor = new WebRTCApplication();
+			try {
+				Class clazz = Class.forName("io.antmedia.enterprise.webrtc.WebRTCApplication");
+				appAdaptor = (AntMediaApplicationAdapter) clazz.newInstance();
+			} catch (Exception e) {
+				logger.error(ExceptionUtils.getStackTrace(e));
```

Comment:
Writing full stack trace may likely frighten the community users that there is a problem in the software. 
Just write a simple information log